patterns/2-structured/innersource-portal.md

```diff
@@ -49,6 +49,7 @@ The InnerSource Portal has enabled InnerSource project owners to advertise their
 * A large financial services organization has used the creation of an InnerSource Portal to provide a mechanism of advertising and discovering InnerSource projects in existence across different business units
 * SAP promotes InnerSource projects in the InnerSource portal - projects can self-register using GitHub topics. The [Repository Activity Score](repository-activity-score.md) defines the default order of the InnerSource projects in the portal.  
 * The InnerSource Portal pattern has been proven to work extremely well with the associated InnerSource Gig Marketplace pattern in this context
+* American Airlines promotes InnerSource projects via an internal InnerSource Marketplace. Similarly to SAP, projects self-register by adding `innersource` as a Github topic. Projects are searchable and filterable by language, topics, number of open issues, etc.
```

Comment:
```suggestion
* American Airlines promotes InnerSource projects via an [internal InnerSource Marketplace](https://tech.aa.com/2020-10-30-innersource/). Similarly to SAP, projects self-register by adding `innersource` as a GitHub topic. Projects are searchable and filterable by language, topics, number of open issues, etc.
* The InnerSource Portal pattern has been proven to work extremely well with the associated InnerSource Gig Marketplace pattern in this context
```

Reply:
Thanks for adding this!

I suggested a couple of changes:

- changed the order of bullet points (so that American Airlines comes directly after the SAP example)
- added a link to your tech blog (while not exclusively talking about the InnerSource Marketplace, it certainly adds some nice color to the story. great job with that blog post too!)

Reply:
Great suggestions! ✅